scripts/shared/cleanup.sh

```diff
@@ -23,7 +21,11 @@ function stop_local_registry {
 
 ### Main ###
 
-run_parallel "{1..3}" delete_cluster
+n=$(kind get clusters | wc -l)
```

Comment:
This is not good, you need to extract the exact names and delete them since there could be gaps for example..

Reply:
This is essentially the same as to what the previous code did except I'm getting the actual #  of clusters instead of hard coding to 3.  That said I agree deleting the actual names retrieved from kind is better although we can't use `run_parallel` but we can still delete in parallel.